raiden/tasks.py

```diff
@@ -8,12 +9,14 @@
 from gevent.queue import Queue
 import structlog
 
+
 from raiden.exceptions import RaidenShuttingDown
 from raiden.utils import get_system_spec
 
 CHECK_VERSION_INTERVAL = 3 * 60 * 60
 LATEST = 'https://api.github.com/repos/raiden-network/raiden/releases/latest'
 RELEASE_PAGE = 'https://github.com/raiden-network/raiden/releases'
+SECURITY_EXPRESSION = '{SECURITY: [a-zA-Z 1-9]*}'
```

Comment:
During the meeting today, we said to use the string "CRITICAL UPDATE" to identify a security/critical update release.

Reply:
I'll change it. Thanks Lefteris.